vuu-ui/packages/vuu-utils/src/date/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { DateTimeColumnTypeSimple } from "packages/vuu-table-types";
```

Comment:
watch out for this `packages` should be @finos

This will have been auto inserted by vs code. Happens to me, though didn't used to. I've tried to fix it by creating tsconfig files in each package. Doesn't seem to have worked. The solution is correct configutation of typescript, so it correctly resolves workspace package references . Haven't got it right yet

Reply:
(It doesn't cause a runtime problem if the import is a type only file, though it would if the file was ts)

Reply:
Thanks for catching that! I can look into helping with the `tsconfig`